build.gradle

```diff
@@ -0,0 +1,223 @@
+                    'org.springframework.boot:spring-boot-starter-data-jpa',
+                    'org.springframework.boot:spring-boot-starter-jdbc',
+                    'org.springframework.boot:spring-boot-starter-log4j2',
+                    //'org.springframework.boot:spring-boot-starter-security'
+                ]
+            ],
+
+            db: [
+                'org.postgresql:postgresql',
+                'org.liquibase:liquibase-core'
+            ],
+            
+            test: [
+                'org.springframework.boot:spring-boot-starter-test'
+            ],            
+
+            misc: [
+                'io.jsonwebtoken:jjwt:0.9.1',
+                'javax.inject:javax.inject:1',
+                'commons-io:commons-io:2.5',
+                'commons-collections:commons-collections:3.2.2',
+                'org.slf4j:slf4j-api'               
+            ]
+        ]
+    }
+
+  
+  	dependencies {
+		classpath "com.netflix.nebula:gradle-lint-plugin:latest.release"
+         
+	}
+    
+}
+	 
+
+plugins {
+    id 'org.springframework.boot' version '2.0.3.RELEASE'
+    id 'org.sonarqube' version '2.6.2'
+    id 'io.spring.dependency-management'  version '1.0.6.RELEASE'
+    id 'eclipse'
+    id 'jacoco'
+    id 'java'
+    id 'checkstyle'
+    id 'pmd'    
+    id 'findbugs'
+    id 'org.owasp.dependencycheck' version '3.3.0'
+    
+}
+
+apply plugin: 'nebula.lint'
+
+dependencyManagement {
+    imports {
+        mavenBom org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES
+    }
+}
+
+group = 'com.navitas'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = 1.8
+
+
+gradleLint {
+  rules  = ['dependency-parentheses']
+  criticalRules = ['unused-dependency'] // <-- this will fail the build in the event of a violation
+}
+
+gradleLint.reportFormat = 'html'
+ 
+  springBoot {
+	 buildInfo()
+
+ }
+
+repositories {
+    mavenLocal()
+	mavenCentral()
+}
+
+configurations {
+    all*.exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
+}
+
+dependencies {
+    compile 'org.springframework:spring-context:5.0.7.RELEASE'
+    compile 'org.springframework.boot:spring-boot:2.0.3.RELEASE'
+  //  runtime 'com.fasterxml.jackson.core:jackson-databind:2.9.6'
+    compile 'org.springframework.boot:spring-boot-autoconfigure:2.0.3.RELEASE'
+    compile 'org.springframework.data:spring-data-jpa:2.0.8.RELEASE'
+//    compile 'org.springframework.security:spring-security-config:5.0.6.RELEASE'
+    compile 'org.springframework:spring-beans:5.0.7.RELEASE'
+//    compile 'org.springframework.security:spring-security-web:5.0.6.RELEASE'
+    compile 'org.hibernate.javax.persistence:hibernate-jpa-2.1-api:1.0.2.Final'
+    compile 'org.springframework.data:spring-data-commons:2.0.8.RELEASE'
+//    compile 'org.springframework.security:spring-security-core:5.0.6.RELEASE'
+    compile 'org.springframework:spring-web:5.0.7.RELEASE'
```

Comment:
What's the point of bringing in all these spring framework dependencies when these are brought in by these jars already? I know it's to satisfy the gradle linter plugin in, but it defeats the purpose of having both the spring boot starters and the individual jars.
```
'org.springframework.boot:spring-boot-starter-web',
'org.springframework.boot:spring-boot-starter-data-jpa',
'org.springframework.boot:spring-boot-starter-jdbc',
'org.springframework.boot:spring-boot-starter-log4j2',
```